packages/supermassive/src/utilities/schemaFragmentFromMinimalViableSchemaDocument.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  DocumentWithMinimalViableSchema,
+  SchemaFragment,
+  UserResolvers,
+} from "../types";
+import { createSchemaDefinitions } from "./mergeSchemaDefinitions";
+
+export function schemaFragmentFromMinimalViableSchemaDocument(
+  document: DocumentWithMinimalViableSchema,
+  resolvers: UserResolvers,
+  operationName?: string,
+): SchemaFragment {
+  const schemaDefinitions = createSchemaDefinitions(
+    document.definitions.map((def) => def.__defs),
+  );
+  const name =
+    operationName ||
+    document.definitions.find((def) => def.name?.value)?.name?.value ||
+    "(anonymous)";
+  return {
+    schemaId: name,
```

Comment:
`schemaId` is needed to cache enum/scalar types per schema (I mean GraphQLEnumType and GraphQLScalarType instances) + for schema loader. If we pass `schemaId` as operation it won't really re-use those instances from the cache. So it was meant to be used with the actual schema id, i.e. "default" / "view schema" in our scenarios